src/tools/convert_dgv.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+
+"""
+Converts existing dgv tab file to a MAVIS format. 
+DGV files can be obtained from: http://dgv.tcag.ca/dgv/app/downloads?ref=GRCh37/hg19
+"""
+
+import argparse
+import logging
+import itertools
+from typing import Dict, Tuple, List
+from mavis.constants import COLUMNS, ORIENT, STRAND, SVTYPE
+import pandas as pd
+from mavis.error import InvalidRearrangement
```

Comment:
imports should be sorted

Reply:
Addressed in [8f1cffb](https://github.com/bcgsc/mavis/pull/336/commits/8f1cffb6e373f828c0cf0c74b796d73bdd3bcd5c)

Reply:
typing is part of the standard python library so its import should be grouped with the first block